draft/accessibility-team.rst

```diff
@@ -0,0 +1,129 @@
+
+Membership
+----------
+
+The team shall not have a fixed size, but instead will grow and shrink
+organically as members choose to leave, and when new members are deemed to be
+required by the rest of the team. Membership of the team works similarly to the
+`Code of Conduct Committe <https://github.com/django/code-of-conduct/blob/master/membership.md>`_.
+New members shall be chosen from a list of volunteers, or if there is a lack
+of volunteers, an advertisement will be published on the Django website.
+Priority will be given to volunteers who, in no particular order:
+
+- Have disabilities that make using the web and/or web development more
+  difficult.
+- Have relevant, positive contributions or expertise in the field.
+- Have a record of contributing to Django.
+
+Members shall remain in the team for a fixed-term of 9 months, after which
+they must opt-in to remain on the team. Membership will also be terminated by:
+
+- Becoming disqualified due to actions taken by the Code of Conduct committe
+  of the Django Software Foundation.
+
+- A vote of the Technical Board, or full consensus of the rest of the
+  Accessibility Team, if the team is considered too large, the person is not
+  making positive contributions, or any other sound reason.
+  (TODO: dunno about any of this)
+
+Responsibilities
+----------------
+
+The remit of the Accessibility Team shall include all user-facing components
+of projects maintained by the Django Software Foundation. This includes but is
+not limited to:
+
+- User-facing parts of Django, such as default HTML output in forms and such,
+  and the django admin and its default theme, HTML, and JavaScript components.
+
+- All websites and other software projects maintained by the DSF, such as the
+  Django website, including documentation, Django Snippets, Django People,
+  the issue tracker, etc.
+
+The responsibilities of the Accessibility Team are expected to change over
+time, and are to be decided by conensus of the team, with input from the
+Technical Board as required. to begin, several areas have been identified:
+
+- Deciding on any relevant accessibility guidelines to follow, such as WCAG,
+  and at which conformance level.
+
+- Implementing automated testing to catch simpler issues, working with the ops
+  team as needed to integrate this into the CI process.
+
+- Fixing accessibility issues in Django and associated projects.
+
+- Reviewing accessibility fixes and other tickets that may affect the
+  accessibility of the project.
+
+To aid in reviewing, the Accessibility Team members shall be added to a team
+in the GitHub organization with read access to relevant repositories, so that
+they may be requested to review pull requests, at the discretion of the author,
+reviewer, or other party.
+
+Many of these duties can be implemented by any contributor, not only by the
+Accessibility Team, however the Accessibility Team exists here as a fallback
+in case nobody else is available to do so.
```

Comment:
I think the team’s role should be much more than a fallback – there’s clearly coordination work needed that you can’t expect any contributor to pick up on and self-organise to do, as well as needing to have a way to make decisions that impact the future of Django, like accessibility standards, testing guidelines, and so on.

Reply:
Yeah. The intention here is just to make it clear that it's expected that contributors will fix issues once they're in the tracker, and not everything _must_ be done by the a11y team. But I agree this wording isn't great, I'll have a think.